internal/oci/container.go

```diff
@@ -357,3 +358,10 @@ func (c *Container) ShouldBeStopped() error {
 	}
 	return nil
 }
+
+func (c *Container) ShouldBeStarted() error {
+	if c.state.Status == ContainerStateCreated {
```

Comment:
Should we check if the container is in running state as well? If not, can we rename the function to `ShouldBeCreated`? `ShouldBeStarted` makes me think that running state is valid as well.